### PR TITLE
Add web api GetNetWorkInfo and change GetPeers

### PR DESCRIPTION
**Pull request by jason-hoopox**

### CODE REVIEW

File: src/AElf.WebApp.Application.Net/NetAppService.cs [hunk 1]
```diff
@@ -44,12 +49,36 @@ public async Task<bool> RemovePeer(string address)
...
         /// </summary>
         /// <returns></returns>
-        public Task<List<string>> GetPeers()
+        public Task<GetNetworkInfoOutput> GetNetworkInfo()
         {
-            return Task.FromResult(_networkService.GetPeerIpList());
+            var output = new GetNetworkInfoOutput
+            {
+                ProtocolVersion = KernelConstants.ProtocolVersion,
+                Version = typeof(NetApplicationWebAppAElfModule).Assembly.GetName().Version.ToString(),
```
Comment (rosona) [inline src/AElf.WebApp.Application.Net/NetAppService.cs]: Assembly.GetName().Version can add cache for perf

File: src/AElf.OS.Core/Network/Application/INetworkService.cs [hunk 2]
```diff
@@ -10,6 +11,7 @@ public interface INetworkService
         Task<bool> AddPeerAsync(string address);
         Task<bool> RemovePeerAsync(string address);
         List<string> GetPeerIpList();
```
Comment (rosona) [inline src/AElf.OS.Core/Network/Application/INetworkService.cs]: Remove if not used by others

File: src/AElf.OS.Core/Network/Application/INetworkService.cs [hunk 3]
```diff
@@ -10,6 +11,7 @@ public interface INetworkService
...
+        List<IPeer> GetPeerList();
```
Comment (rosona) [inline src/AElf.OS.Core/Network/Application/INetworkService.cs]: Rname to GetPeers?

File: src/AElf.OS.Core/Network/Application/INetworkService.cs (hunk 2 above)
Comment (jason-hoopox) [reply src/AElf.OS.Core/Network/Application/INetworkService.cs]: It is still used by rpc.

**Pull request merged**